File: Java/nastyHacks.java
import java.util.Scanner;

/**
 * Created by August DH on 2017-09-15.
 */
public class nastyHacks {
    public static void advertise(String input){
        String[] splitted = input.split(" ");
        int r = Integer.parseInt(splitted[0]);
        int e = Integer.parseInt(splitted[1]);
        int c = Integer.parseInt(splitted[2]);

        int adRev = e - c;

        if(r > adRev){
            System.out.println("do not advertise");
        }
        else if(r == adRev){
            System.out.println("does not matter");
        }
        else{
            System.out.println("advertise");
        }
    }

    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);
        int cases = sc.nextInt();
        sc.nextLine();

        for(int i = 0; i < cases; i++){
            String input = sc.nextLine();
            advertise(input);
        }
    }
}

File: Javascript (NODE JS)/nastyhacks.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

size = 0

rl.on('line', (line) => {
    if(size == 0){
        size = parseInt(line)
    }
    else{
        splitted_str = line.split(" ")
        rev = parseInt(splitted_str[0])
        e = parseInt(splitted_str[1])
        c = parseInt(splitted_str[2])
        ad_rev = e-c
        if(ad_rev > rev){
            console.log("advertise")
        }
        else if(ad_rev == rev){
            console.log("does not matter")
        }
        else{
            console.log("do not advertise")
        }
    }
});
